includes/Modules/PageSpeed_Insights.php

```diff
@@ -47,12 +46,11 @@ public function register() {}
 	 * @return bool True if module is connected, false otherwise.
 	 */
 	public function is_connected() {
-		$api_key = $this->authentication->get_api_key_client()->get_api_key();
-		if ( empty( $api_key ) ) {
-			return false;
-		}
-
-		return parent::is_connected();
+		return in_array(
```

Comment:
This method is not for checking whether the scopes are already granted, it's only for additional requirements (see other modules). Hence, this method should be removed from here (the base class implementation simply returns true).